index.html

```diff
@@ -4,6 +4,7 @@
         {{> css}}
         {{> scripts}}
         {{> head}}
+        <meta charset="UTF-8">
```

Comment:
Without this, special characters are not preserved in my test case.

Reply:
I had the same issue. I saw it in FontAwesome rendering strange characters instead of Unicode.